gsoc/background.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title:  "Background on GSoC: start here!"
+show_main: false 
+---
+
+# Background: Start Here!
+
+### What is GSoC? 
+
+Google Summer of Code is a remote summer coding internship program funded by 
+Google. Google distributes funds to open software development organizations to 
+recruit new developers and mentors for summer internships. You get paid to write 
+software at home (or wherever you like), with guidance from expert mentors 
+distributed around the globe.
+
+To apply to be a GSoC intern, you must write an application to GSoC and an 
+application to OpenAstronomy. Each summer, several projects are offered by 
+potential OpenAstronomy mentors, with well-defined scope and pre-requisite 
+experience. If you are selected by both Google and OpenAstronomy, you will be
+a GSoC intern.
+
+### What is OpenAstronomy?
+
+OpenAstronomy is an umbrella organization which acts as an interface between
+the Google Summer of Code and several open source, open development projects
+in the astronomical community, including astropy and sunpy. It coordinates
+mentors and projects for Google Summer of Code internships related to astronomy.
+
+### What is the application process?
+
+You must submit your application to the Google Summer of Code in two parts -- 
+one for Google, and one for your potential project mentor. There's some basic 
+biographical information that Google needs for their part of the application, 
+and we will ask you for more project-specific personal backaground information 
+in the OpenAstronomy application. 
```

Comment:
I would just say to fill up the form, without getting into details who is asking what...